modules/fs2/src/smithy4s/fs2lib/package.scala

```diff
@@ -0,0 +1,12 @@
+package smithy4s
+
+import _root_.fs2.Stream
+
+package object fs2lib {
+
+  type XmlByteStreamEncoder[F[_], A] =
+    smithy4s.codecs.Writer[Any, Stream[F, Byte], A]
+  type XmlByteStreamDecoder[F[_], A] =
```

Comment:
let's call those `ByteStreamEncoder` and `ByteStreamDecoder`, since they don't need to be XML specific